src/game/shared/tf/tf_weaponbase_melee.cpp

```diff
@@ -271,6 +271,11 @@ void CTFWeaponBaseMelee::PlaySwingSound( void )
 //-----------------------------------------------------------------------------
 void CTFWeaponBaseMelee::Swing( CTFPlayer *pPlayer )
 {
+	if ( m_flSmackTime>=m_flNextPrimaryAttack )
```

Comment:
Add spaces around the operator, and also add a comment mentioning why this exists